internal/controller/volsync/vshandler.go

```diff
@@ -207,6 +211,77 @@ func RDStatusReady(rd *volsyncv1alpha1.ReplicationDestination, log logr.Logger)
 	return true
 }
 
+func (v *VSHandler) setRDAsOwnerOfPVC(
+	rd *volsyncv1alpha1.ReplicationDestination,
+	pvc *corev1.PersistentVolumeClaim,
+) error {
+	// Only set OwnerReference if RD and PVC are in the same namespace
```

Comment:
This explains what the code does which is pretty clear. It should explain why ownership is needed only when the pvc and rd are in the same namespace.